src/Gold.Redis/Gold.Redis.Common/Models/Configuration/RedisConnectionConfiguration.cs

```diff
@@ -6,8 +6,9 @@ namespace Gold.Redis.Common.Models.Configuration
 {
     public class RedisConnectionConfiguration
     {
-        public string Host { get; set; }
+        public string Host { get; set; } = "localhost";
```

Comment:
I dont like it. I will fix that later.

Reply:
what why you don't like it its default options its totally common thing to do